plugins/jobs/web_client/views/timeChartConfig.js

```diff
@@ -270,7 +270,7 @@ export default {
     'legends': [
         {
             'fill': 'timing',
```

Comment:
@brianhelba no legend or legend icon? do you see anything different from the gif https://github.com/girder/girder/pull/1901